### PR TITLE
Add camunda-modeler.app v1.2.0

### PR DESCRIPTION
**Pull request by jangalinski**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

camunda-modeler is an open source modelling tool for business processes (bpmn, cmmn, dmn)
- initial setup for 1.2.0

### CODE REVIEW

**Comment by vitorgalvao** [comment]
You’re adding a new cask, not editing an existing one.

Also, as the instructions says, you’re supposed to delete what you do not need.

File: Casks/camunda-modeler.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'camunda-modeler' do
+  version '1.2.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/camunda-modeler.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/camunda-modeler.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://camunda.org/release/camunda-modeler/#{version}/camunda-modeler-#{version}-darwin-x64.tar.gz"
+  name 'camunda-modeler'
```
Comment (vitorgalvao) [inline Casks/camunda-modeler.rb]: `name 'Camunda Modeler'`

**Comment by jangalinski** [comment]
I somehow broke my local repo (had to upgrade cask). I will provide a new pr

**Pull request closed without merging**